www/src/components/SocialIcon.astro

```diff
@@ -0,0 +1,122 @@
+---
+export interface Props {
+  url: string;
+}
+
+const { url, ...props } = Astro.props;
+
+type Icon = 'github' | 'opencollective' | 'linkedin' | 'x' | 'w3c' | 'bluesky';
+
+let icon: Icon | undefined;
+
+if (url.includes('bluesky.app') || url.includes('bsky.app')) {
```

Comment:
URL sniffing is very dumb; it was only out of laziness to reuse the existing nav links without adding another property